middleware/requestid/config.go

```diff
@@ -57,7 +57,7 @@ func configDefault(config ...Config) Config {
 	if cfg.Generator == nil {
 		cfg.Generator = ConfigDefault.Generator
 	}
-	if cfg.ContextKey == "" {
+	if cfg.ContextKey == nil || cfg.ContextKey == "" {
```

Comment:
ContextKey will only ever be "" if the user explicitly sets it to that value. In that case, overriding it to be "requestid" might be surprising.
Though I suppose changing it to accept "" now might break someone's code out there.

Reply:
should be fine, check this https://github.com/gofiber/fiber/pull/2731#issuecomment-1823236098